docs/tutorials/flight_controller_telemetry_tutorial.rst

```diff
@@ -0,0 +1,395 @@
+
+To convert from ERPM to RPM, the ERPM value should be divided by the pole count of the motor divided by 2. The flight controller is expecting to receive 
+ERPM/100, so when it receives telemetry it tries to calculate RPM using the following formula shown below. This calculation can also be
+seen in the `ArduCopter code for BLHeli telemetry parsing <https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_BLHeli/AP_BLHeli.cpp#L1447>`_.
+
+.. figure:: ../_static/tutorial_images/fc_telemetry_tutorial/px4_erpm_to_rpm_formula.png
+    :align: center
+    :width: 40%
+    :alt: ERPM to RPM Conversion
+
+    ERPM to RPM Conversion
+
+Because Vertiq modules send RPM instead of ERPM, this conversion is unnecessary, and actually leads to the flight controller calculating the incorrect RPM by default.
+
+As discussed in the :ref:`fc_telemetry_erpm_to_rpm` section for PX4 below, it is possible to workaround this conversion issue by setting the number of motor poles to 200.
+However, as of v4.3.6, ArduCopter flight controllers limit the maximum value of SERVO_BLH_POLES to 127. So it is not possible to apply this workaround when using
+ArduCopter. The RPM value calculated by the flight controller will not be the true RPM value of the module because of this, but any logged telemetry data can be converted
+based on the formula above to get the actual RPM value.
+
+Standard PWM Telemetry
+***********************
+Vertiq modules support sending ESC telemetry when using :ref:`Standard PWM <hobby_standard_pwm>` or :ref:`other analog protocols <hobby_other_protocols>`
+as covered in the :ref:`ESC Telemetry <telemetry_analog_request>` section of the Feature Reference Manual. This telemetry is requested by sending a 
+:ref:`30 microsecond pulse <telemetry_analog_request>` to the module over the throttle line.  
+
+However, as of version 4.3.6 ArduCopter flight controllers do not seem to support this method of requesting ESC telemetry according to 
+the `ArduCopter documentation (see the note on polling for telemetry data on non-DSHOT protocols) <https://ardupilot.org/copter/docs/common-blheli32-passthru.html#esc-telemetry>`_.
+The relevant note from the ArduCopter documentation is shown below.
+
+.. figure:: ../_static/tutorial_images/fc_telemetry_tutorial/arducopter_no_pwm_telem.png
+    :align: center
+    :width: 60%
+    :alt: ArduCopter Note on ESC Telemetry for non-DSHOT Protocols
+
+    ArduCopter Note on ESC Telemetry for non-DSHOT Protocols
+
+DroneCAN Telemetry
+******************
+For configuring your module, follow the module set up steps in the :ref:`tutorial for integrating a Vertiq module using DroneCAN with a PX4 flight controller <dronecan_fc_tutorial>`.
+For flight controller set up, refer to the ArduCopter documentation on setting up `DroneCAN <https://ardupilot.org/copter/docs/common-uavcan-setup-advanced.html>`_ 

```

Comment:
setup not set up

Reply:
Which part is this referring to? there are several instances of set up here, is it all of them? The flight controller one? The module one?